lib/util.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const url = require('url');
+
+class Util {
+    timeDiff(from, to, digits) {
+        if (digits === undefined) { digits = 3; }
+        let ms = (to[0] - from[0]) * 1e3
+            + (to[1] - from[1]) * 1e-6;
+        return +ms.toFixed(digits);
+    }
+
+    urlKeys() {
+        Object.keys(url.parse(''));
+    }
+}
+
+module.exports = new Util();
```

Comment:
This doesn't doesn't need to be a class, this would be a collection of functions.
urlKeys should be a constant. It can be a constant value of this files returned object and still tested.